### PR TITLE
fixed styling in code snippet

### PR DESCRIPTION
**Pull request by blentz100**
In the Course Overview + the shell page, there is a code snippet `man ls` that had styling that didn't match the other snippets. Changed the styling in the html to match the other snippets. Not sure if this is useful or not since it looks like the html is autogenerated when the web server is launched? If this isn't the best way to submit a PR for this, would love to hear how to do so? Thanks.

### CODE REVIEW

File: _site/2020/course-shell/index.html [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+rename/move a file), <code class="language-plaintext highlighter-rouge">cp</code> (to copy a file), and <code class="language-plaintext highlighter-rouge">mkdir</code> (to make a new
+directory).</p>
+
+<p>If you ever want <em>more</em> information about a program’s arguments, inputs,
+outputs, or how it works in general, give the <code class="language-plaintext highlighter-rouge">man</code> program a try. It
+takes as an argument the name of a program, and shows you its <em>manual
+page</em>. Press <code class="language-plaintext highlighter-rouge">q</code> to exit.</p>
+
+<div class="language-console highlighter-rouge"><div class="highlight"><pre
+class="highlight"><code><span class="gp">missing:~$</span><span class="w"> </span><span class="nb">man</span> ls
```
Comment (blentz100) [inline _site/2020/course-shell/index.html:302]: This was the particular line that needed the styling change. Applied class "nb" to "man" and removed styling from "ls"

**Comment by anishathalye** [comment]
Thanks for reporting this. As you point out, we can't modify contents in `_site` because it's auto-generated by Jekyll from the markdown and templates. That page is generated from https://github.com/missing-semester/missing-semester/blob/master/_2020/course-shell.md.

The relevant snippet is:

    ```console
    missing:~$ man ls
    ```

I guess the automatic syntax highlighting setup doesn't handle it correctly. But I don't think it's worth figuring out how to special-case this one thing.

**Comment by jonhoo** [comment]
For console blocks, I generally skip the host and path, and then it looks okay:
```console
$ man ls
```

**Comment by anishathalye** [comment]
Most of the examples shown are fine, with or without the host/path, e.g.:

Screen Shot 2021-04-23 at 2 54 50 PM

I think whatever syntax highlighter we are using has some special handling for `man`:

Screen Shot 2021-04-23 at 2 54 54 PM

It looks like this whether the prompt is the fancy one shown or just `$`.

**Pull request closed without merging**